src/applications/hca/tests/containers/ConfirmationPage.unit.spec.jsx

```diff
@@ -26,7 +26,7 @@ describe('hca <ConfirmationPage>', () => {
     const tree = SkinDeep.shallowRender(<ConfirmationPage form={form} />);
 
     expect(tree.subTree('.claim-list')).to.exist;
-    expect(tree.everySubTree('span')[2].text()).to.contain('Jan. 1, 2010');
+    expect(tree.everySubTree('span')[2].text()).to.contain('Jan 1, 2010');
```

Comment:
The formatting for abbreviated months in `date-fns` does not include a period for truncated months.

Reply:
The [content style guide](https://design.va.gov/content-style-guide/dates-and-numbers#dates-and-years) indicates they have to have a period. I [think](https://date-fns.org/v2.11.1/docs/format#examples) we can use the escape character to add a period in the format string. Maybe 

```
{format(parseISO(response.timestamp), "MMM'.' d, yyyy")}
```

Reply:
I'll mess around with this - we may have some edge cases to account for with shorter months (i.e. `March. 14, 2020`), so I'll see how it handles those.

Reply:
That true - we aren't supposed to abbreviate those. 
![external-content duckduckgo-9](https://user-images.githubusercontent.com/4998130/78580890-17ee1200-7801-11ea-9256-3f189b314f43.jpg)

Reply:
I've seen some code that Eugene wrote to handle this exact thing for downtime notifications.

Reply:
[Closing for now](https://github.com/department-of-veterans-affairs/va.gov-team/issues/6763#issuecomment-609983881).  The extra work to ensure that _every_ format call meets the content guidelines isn't worth it in my opinion. I'd like for `date-fns` to provide a way of changing the default locale so we have an easier way of controlling things like this.